.env.schema

```diff
@@ -28,6 +28,10 @@ NEXT_PUBLIC_EGO_API_URL=
 NEXT_PUBLIC_EGO_CLIENT_ID=
 NEXT_PUBLIC_KEYCLOAK_API_URL=
 
+# ###### Environmental submission
+NEXT_PUBLIC_ENVIRONMENTAL_SUBMISSION_API_URL=
+NEXT_PUBLIC_ENVIRONMENTAL_SUBMISSION_CATEGORY_ID=
```

Comment:
environment variables to set up Submission service

Reply:
what's a category id in this context?